### ISSUE TITLE
Client Creation - Image Controls

### ISSUE THREAD

**Issue by muirdata** [muirdata/muir-data#338]
In Dev, the control for logo image size appear right away before the drag and drop box shows up. These should not show until drag and drop and browse upload folder icon shows. This typically happens after the form is filled out to a minimum level. See Production. The logo size control is an enhancement and could be causing problems.

![image](https://cloud.githubusercontent.com/assets/2980580/24324239/0be30072-1140-11e7-91aa-764d617dcc06.png)

**Comment by JohnKJohansen-MDS** [comment]
The actual problem is that in Client Control, once you pause in typing the name, it is creating the customer, ignoring the rest of the required fields. If you type more in the name, it creates a second customer. By the time I finished typing the name of a customer in dev, I had three client records.

**Comment by muirdata** [comment]
Yes, that sounds right. It should not create only one Client. The Client should not be created until all required (*) fields are defined.

**Comment by JohnKJohansen-MDS** [comment]
Limited Logo size to after initial client creation

![image](https://cloud.githubusercontent.com/assets/24964880/24328268/be5c9ba6-1199-11e7-91e3-1e7ab019dc0d.png)

**Comment by muirdata** [comment]
Looks good. Will test when it becomes available.

**Comment by muirdata** [comment]
I think we'll need to discuss this issue in more detail. We need to figure out an efficient way to be able to deal with different logo shapes and size in the report header. The original intent of these controls was to be able to change the size of the logo.

**Comment by JohnKJohansen-MDS** [comment]
OK. I think we should have a plan for the following cases
1. The uploaded log is very close to our target size
2. The logo is too deep for our target size, but the width is close to our target size
3. The logo is to wide, but the depth is about right
4. The logo is to small for our target size
5. the logo is too large, but the ratio of width to height is close
6. the logo is too large, and the ratio is wrong

Our options are to 
1. Create rules based on the above cases, and dynamically resize
2. Give the client access to an image editor, and let them decide how best to fix their logo, resist accepting the logo until it meets our minimum size and ratio requirements
3. Provide the client with our logo size limits, and ask them to load images of that size: auto crop rezie the image, maintaining aspect ratio, until height and width fit in our header window, then left justify it in the header.

**Comment by JohnKJohansen-MDS** [comment]
There is nothing more here for me to fix related to the original bug. Product design for how we deal with images is a separate issue.
